### ISSUE TITLE
During the training the loss is nan,how to solve it?

### ISSUE THREAD

**Issue by LDoubleZhi** [facebookresearch/maskrcnn-benchmark#283]
## ❓ Questions and Help
  0.3550 (0.3499)  data: 0.0100 (0.0095)  lr: 0.016773  max mem: 3050
2018-12-18 10:21:38,024 maskrcnn_benchmark.trainer INFO: eta: 5:30:07  iter: 400  loss: 0.2376 (0.2817)  loss_classifier: 0.1325 (0.1725)  loss_box_reg: 0.0580 (0.0655)  loss_objectness: 0.0156 (0.0368)  loss_rpn_box_reg: 0.0078 (0.0070)  time: 0.3508 (0.3500)  data: 0.0095 (0.0094)  lr: 0.017307  max mem: 3050
2018-12-18 10:21:45,031 maskrcnn_benchmark.trainer INFO: eta: 5:30:01  iter: 420  loss: 0.2443 (0.2846)  loss_classifier: 0.1486 (0.1752)  loss_box_reg: 0.0648 (0.0653)  loss_objectness: 0.0243 (0.0369)  loss_rpn_box_reg: 0.0052 (0.0072)  time: 0.3503 (0.3500)  data: 0.0072 (0.0094)  lr: 0.017840  max mem: 3050
2018-12-18 10:21:51,975 maskrcnn_benchmark.trainer INFO: eta: 5:29:47  iter: 440  loss: 0.3062 (0.2921)  loss_classifier: 0.1738 (0.1820)  loss_box_reg: 0.0553 (0.0657)  loss_objectness: 0.0246 (0.0370)  loss_rpn_box_reg: 0.0061 (0.0074)  time: 0.3447 (0.3498)  data: 0.0053 (0.0092)  lr: 0.018373  max mem: 3050
2018-12-18 10:21:58,887 maskrcnn_benchmark.trainer INFO: eta: 5:29:29  iter: 460  loss: 0.2209 (0.2948)  loss_classifier: 0.1323 (0.1839)  loss_box_reg: 0.0235 (0.0644)  loss_objectness: 0.0330 (0.0389)  loss_rpn_box_reg: 0.0064 (0.0077)  time: 0.3444 (0.3497)  data: 0.0049 (0.0090)  lr: 0.018907  max mem: 3050
2018-12-18 10:22:05,756 maskrcnn_benchmark.trainer INFO: eta: 5:29:08  iter: 480  loss: 0.2156 (nan)  loss_classifier: 0.2242 (nan)  loss_box_reg: nan (nan)  loss_objectness: 0.1445 (nan)  loss_rpn_box_reg: 0.0161 (inf)  time: 0.3445 (0.3494)  data: 0.0051 (0.0089)  lr: 0.019440  max mem: 3050
2018-12-18 10:22:12,212 maskrcnn_benchmark.trainer INFO: eta: 5:28:01  iter: 500  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: 0.0000 (nan)  time: 0.3226 (0.3483)  data: 0.0051 (0.0087)  lr: 0.019973  max mem: 3050
2018-12-18 10:22:18,680 maskrcnn_benchmark.trainer INFO: eta: 5:27:00  iter: 520  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: nan (nan)  time: 0.3232 (0.3474)  data: 0.0050 (0.0086)  lr: 0.020000  max mem: 3050
2018-12-18 10:22:25,151 maskrcnn_benchmark.trainer INFO: eta: 5:26:03  iter: 540  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: nan (nan)  time: 0.3233 (0.3465)  data: 0.0051 (0.0085)  lr: 0.020000  max mem: 3050
2018-12-18 10:22:31,626 maskrcnn_benchmark.trainer INFO: eta: 5:25:10  iter: 560  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: 0.0000 (nan)  time: 0.3236 (0.3457)  data: 0.0053 (0.0084)  lr: 0.020000  max mem: 3050
2018-12-18 10:22:38,101 maskrcnn_benchmark.trainer INFO: eta: 5:24:20  iter: 580  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: nan (nan)  time: 0.3237 (0.3449)  data: 0.0051 (0.0082)  lr: 0.020000  max mem: 3050
2018-12-18 10:22:44,571 maskrcnn_benchmark.trainer INFO: eta: 5:23:33  iter: 600  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: nan (nan)  time: 0.3233 (0.3442)  data: 0.0051 (0.0081)  lr: 0.020000  max mem: 3050
2018-12-18 10:22:51,057 maskrcnn_benchmark.trainer INFO: eta: 5:22:50  iter: 620  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: 0.0000 (nan)  time: 0.3242 (0.3436)  data: 0.0053 (0.0080)  lr: 0.020000  max mem: 3050
2018-12-18 10:22:57,535 maskrcnn_benchmark.trainer INFO: eta: 5:22:09  iter: 640  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: 0.0000 (nan)  time: 0.3235 (0.3430)  data: 0.0051 (0.0080)  lr: 0.020000  max mem: 3050
2018-12-18 10:23:04,009 maskrcnn_benchmark.trainer INFO: eta: 5:21:29  iter: 660  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: 0.0000 (nan)  time: 0.3235 (0.3424)  data: 0.0051 (0.0079)  lr: 0.020000  max mem: 3050
2018-12-18 10:23:10,498 maskrcnn_benchmark.trainer INFO: eta: 5:20:52  iter: 680  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: nan (nan)  time: 0.3237 (0.3418)  data: 0.0053 (0.0078)  lr: 0.020000  max mem: 3050
2018-12-18 10:23:16,969 maskrcnn_benchmark.trainer INFO: eta: 5:20:16  iter: 700  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: nan (nan)  time: 0.3235 (0.3413)  data: 0.0052 (0.0077)  lr: 0.020000  max mem: 3050
2018-12-18 10:23:23,444 maskrcnn_benchmark.trainer INFO: eta: 5:19:42  iter: 720  loss: nan (nan)  loss_classifier: nan (nan)  loss_box_reg: nan (nan)  loss_objectness: nan (nan)  loss_rpn_box_reg: nan (nan)  time: 0.3237 (0.3408)  data: 0.0053 (0.0077)  lr: 0.020000  max mem: 3050

**Comment by LDoubleZhi** [comment]
well，I reduce the lr, and it worked.

**Comment by fmassa** [comment]
Yes, reducing the learning rate usually helps in those cases.
